### DISCUSSION TITLE
Entry in CTk

### DISCUSSION THREAD

**Discussion by TiagoHiray** [TomSchimansky/CustomTkinter#1663] [Q&A] [answered]
Hey guys, I'm currenty doing a uni project and using ctk. I'm trying to make a page with an entry field with a height of 200 pixels. However, when you type in this, the entry text begins to be written in the middle of the box and the text does not get broken. Does anyone know how to make the text start in the top of the box and if there is a word break function in ctk??
![githubctkdiscussao](https://github.com/TomSchimansky/CustomTkinter/assets/104155224/82d870f0-1802-4a1f-9a85-475512016690)

**Comment by zSynctic** [comment, marked as answer]
You can replace the entry widget with a textbox widget since a entry allows only one line of text while the textbox widget allows multiple lines

**Reply by TiagoHiray** [reply]
Yeah, I noticed it after a while. Thanks!
